README.md

```diff
@@ -141,17 +141,19 @@ String (Default: amd64)
 Specify architecture. Architecture other than the default amd64 can be specified with the `TFENV_ARCH` environment variable
 
 ```console
-TFENV_ARCH=arm tfenv install 0.7.9
+$ TFENV_ARCH=arm
+$ tfenv install 0.7.9
 ```
 
 ##### `TFENV_AUTO_INSTALL`
 
-String (Default: true)
+Boolean (Default: true)
 
 Should tfenv automatically install terraform if the version specified by defaults or a .terraform-version file is not currently installed.
 
 ```console
-TFENV_AUTO_INSTALL=false terraform plan
+$ TFENV_AUTO_INSTALL=false
+$ terraform plan
```

Comment:
It's entirely normal to have environment variables set as an override for a singular command